### PR TITLE
Add dockerfile

### PR DESCRIPTION
**Pull request by jackzampolin**
@kantai Was adding browser to the `blockstack-core` `Dockerfile` so I've gone ahead and split that work off to this repo. We are pushing these images to `quay.io` to make them publicly available. This will be pulled down by the `blockstack-core` install. 

@larrysalibra I need a hand understanding how to run (What ports need to be exposed) and how to configure (point the browser at another docker container running the API) the `apt` install. I'm not seeing that in the README. Holler at me.

### CODE REVIEW

**Comment by jackzampolin** [comment]
Related to https://github.com/blockstack/blockstack-core/pull/572

**Comment by jackzampolin** [comment]
Per @larrysalibra:

- [ ] Expose CORS Proxy running on `localhost:1337`
- [ ] Expose static file server running on `localohost:8888`
- [ ] May have to manually connect this to the `blockstack api` instance running in the other container.

**Comment by jackzampolin** [comment]
This is working with the core install. I've also added a CLI to wrap the standard operations. I think this is ready to merge @larrysalibra

**Comment by larrysalibra** [comment]
Awesome @jackzampolin ! We try to avoid the prefix "bs" for hopefully obvious reasons. What do you think about renaming it to `blockstack-docker.sh` or something like that?

**Comment by jackzampolin** [comment]
@larrysalibra Ok. I had a feeling that was the case. Ideally wanted something that was a bit easier to type. How do you feel about `launcher` (like discourse) or something similar?

**Comment by jackzampolin** [comment]
@larrysalibra I've gone ahead and added the Core stuff to this browser launcher as well.

File: launcher [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+  
+  # Use init containers to set the API bind to 0.0.0.0
+  docker run -it -v $homedir:$containerdir $coreimage sed -i 's/api_endpoint_bind = localhost/api_endpoint_bind = 0.0.0.0/' $containerdir/client.in -e CORSPROXY_HOST="0.0.0.0"i
+  docker run -it -v $homedir:$containerdir $coreimage sed -i 's/api_endpoint_host = localhost/api_endpoint_host = 0.0.0.0/' $containerdir/client.ini
+}
+
+start () {
+  # Check for args first
+  if [ $# -eq 0 ]; then
+    echo "Need to input password for wallet located in the $HOME/.blockstack folder when staring api: ./launcher start mypass"
```
Comment (larrysalibra) [inline launcher]: @jackzampolin typos

**Pull request merged**